mne_bids/mne_bids.py

```diff
@@ -753,11 +753,20 @@ def write_raw_bids(raw, bids_basename, output_path, events_data=None,
     # KIT data requires the marker file to be copied over too
     if 'mrk' in raw._init_kwargs:
         hpi = raw._init_kwargs['mrk']
-        _, marker_ext = _parse_ext(hpi)
-        marker_fname = make_bids_basename(
-            subject=subject_id, session=session_id, task=task, run=run,
-            acquisition=acquisition, suffix='markers%s' % marker_ext,
-            prefix=os.path.join(data_path, bids_raw_folder))
-        sh.copyfile(hpi, marker_fname)
+        acq_map = dict()
+        if isinstance(hpi, list):
+            # order hpi list by acquisition time
+            hpi.sort(key=_get_mrk_meas_date)
```

Comment:
what happens if `meas_date` is `None` ? 

do we get an error?

Reply:
it retains the order of the list. I realised earlier that I should have commented somewhere in the docs that you can pass multiple markers and if you do they will be given the `pre` and `post` tags, unless they haven't got time stamps, in which case they will be `pre` and `post` in the order they are in the list.

Reply:
so the `pre` and `post` tags could potentially be wrong in that case? sounds dangerous to me ...

Reply:
well if you specify that they need to be provided in the right order if the date has somehow been removed, then it is on the user. I don't know how common it would be that the marker files have their dates removed. I am not even sure if it is something you can easily do...

Reply:
But the user cannot provide `hpi` at the moment ... it's read automatically and we don't recommend messing with private attributes. Hmm ... I understand this PR better now. It's a clever solution what you've done. But we need to be careful in the case that `meas_date` is not known.

Reply:
They can indirectly though. When they generate the `raw` object they pass the markers into the `hpi` argument of `read_raw_kit`, so they have control over the order of the markers *and* have to specify the markers anyway.

Reply:
I see, okay that works. But we should carefully document it. Maybe a tiny example? Btw, your test violates this principle that you mentioned so you need to fix it.

Reply:
any suggestions on where to put an example?
We don't have any examples for KIT stuff, only elekta...